jme3-desktop/src/main/java/com/jme3/input/AWTKeyInput.java

```
@@ -234,4 +234,8 @@ public void keyReleased(KeyEvent e) {
       System.out.println("Key released "+e.getKeyChar());
       onKeyEvent(e, false);
     }
+
+    public String getKeyName(int key){
+      return null;
```

might be better to throw an UnsupportedOperationException

---

Still needs resolution.